postgres-types/src/lib.rs

```diff
@@ -769,6 +769,40 @@ pub trait ToSql: fmt::Debug {
         ty: &Type,
         out: &mut BytesMut,
     ) -> Result<IsNull, Box<dyn Error + Sync + Send>>;
+
+    /// Specify the encode format
+    fn encode_format(&self) -> i16 { 1 }
```

Comment:
This should use an enum rather than a raw integer.

Reply:
I'll look into it and fix it.

Reply:
Pardon the question (lack of knowledge), what kind of enum?
I looked over [here](https://github.com/sfackler/rust-postgres/blob/master/postgres-protocol/src/message/frontend.rs#L59) and the function seems to expect `I: IntoIterator<Item = i16>` 

I'll change to whatever you think it's best

Reply:
You can make whatever `Format` you'd like and then add a `From` impl for `i16`. Here's what I did over on my fork -> https://github.com/NAlexPear/rust-postgres/blob/d8d4264786204bb1ef9d54d2fa61460674917514/postgres-types/src/lib.rs#L791